river/Config.zig

```diff
@@ -156,3 +165,19 @@ pub fn shouldFloat(self: Self, view: *View) bool {
 
     return false;
 }
+
+pub fn csdAllowed(self: Self, toplevel: *wlr.XdgToplevel) bool {
```

Comment:
The implementation differs from the one for `filter`s, however, because I did not find a way to get a `View` from a `Decoration`: Instead, we pass `*wlr.XdgToplevel` around. I'm not sure whether this is neccessary or there is a better approach.

Reply:
I've pushed a commit which uses `*View`s, but thereby introducing additional complexity at the call site in `Decoration.zig`. I use the same mechanism as is used in `csdFilterUpdateViews`, let me know if this is better/worse than the above alternative.